dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -51,6 +51,7 @@ commons-math3/3.6.1//commons-math3-3.6.1.jar
 commons-pool/1.5.4//commons-pool-1.5.4.jar
 commons-text/1.10.0//commons-text-1.10.0.jar
 compress-lzf/1.1.2//compress-lzf-1.1.2.jar
+cos_api-bundle/5.6.69//cos_api-bundle-5.6.69.jar
```

Comment:
BTW, is this introduced back?

Reply:
yes, but its the version with the updated suffix list. https://github.com/apache/hadoop/pull/4444

Reply:
It's a little magical, https://github.com/apache/hadoop/pull/4444 is a version upgrade. I think it should be easier to understand when similar

```
cos_api-bundle/5.6.19//cos_api-bundle-5.6.19.jar  -> cos_api-bundle/5.6.69//cos_api-bundle-5.6.69.jar
```
changes occur, but when using Hadoop 3.3.4, this dependency does not appear in `spark-deps-hadoop-3-hive-2.3`

Reply:
Is this caused by https://github.com/apache/spark/pull/39124#discussion_r1146542183?